browser_patches/webkit/patches/bootstrap.diff

```diff
@@ -3423,22 +3418,19 @@ index 864ce3cfe13994642495f2d7dfbdcc9a4cad9f24..50a4c389709d92a251214b24ba6068d9
  #include "ScriptState.h"
  #include "SecurityOrigin.h"
  #include "UserGestureEmulationScope.h"
-@@ -103,9 +105,25 @@ void PageRuntimeAgent::didClearWindowObjectInWorld(Frame& frame)
-     if (!pageAgent)
+@@ -104,6 +106,24 @@ void PageRuntimeAgent::didClearWindowObjectInWorld(Frame& frame, DOMWrapperWorld
          return;
  
--    auto frameId = pageAgent->frameId(&frame);
-+    // Force main world creation.
-     auto* scriptState = mainWorldExecState(&frame);
--    notifyContextCreated(frameId, scriptState, nullptr, true);
-+    Document* document = frame.document();
-+    String worldName = document ? document->securityOrigin().toRawString() : String();
-+    notifyContextCreated(frame, scriptState, worldName, true);
-+    frame.script().evaluateIgnoringException(ScriptSourceCode(pageAgent->bootstrapScript(), URL { URL(), "web-inspector://bootstrap.js"_s }));
+     notifyContextCreated(pageAgent->frameId(&frame), frame.script().globalObject(world), world);
 +
-+    // Create user worlds.
-+    for (auto pair : pageAgent->isolatedWorldScripts())
-+        createIsolatedWorld(frame, pair.key, pair.value);
```

Comment:
This code used to force creation of the isolated worlds for new frames. The new code will only initialize it if already exists. We need a code that ensures isolated worlds creation for new frames.

Reply:
Worlds outlive navigation, notifyCreated will be called and it'll create our isolated world.